operator/src/main/java/oracle/kubernetes/operator/EventConstants.java

```diff
@@ -78,4 +80,5 @@
   String SERVER_POD_ERROR_SUGGESTION = "";
   String REPLICAS_TOO_HIGH_ERROR_SUGGESTION = "Lower replicas in the domain resource, or increase the number "
       + "of WebLogic servers in the WebLogic domain configuration for the cluster.";
+  String DOMAIN_CONVERSION_FAILED_PATTERN = "WebLogic Domain custom resource conversion webhook failed due to '%s' %s";
```

Comment:
Why is this here? You can access the string from the resource bundle and format against it without having a copy in the code.

Reply:
I was following same pattern as other messages such as `REPLICAS_TOO_HIGH_ERROR_SUGGESTION `, I have removed it.